package-system/Qt/build-linux-x86.sh

```diff
@@ -11,7 +11,8 @@
 
 TIFF_PACKAGE=tiff-4.2.0.15-rev3-linux
 ZLIB_PACKAGE=zlib-1.2.11-rev5-linux
+OPENSSL_PACKAGE=OpenSSL-1.1.1t-rev1-linux
```

Comment:
Is there a way that we can get these set as environment variables using the build_config.json, so that we don't have the package revision in scripts.?

Reply:
Not without more changes to `pull_and_build_from_git`. These packages are defined using the `depends_on_packages` attribute in the build_config.json, which would require the addition of an additional 'KEY' field, and then all projects will need to update using this key. I don't mind doing this, but i want to reduce the scope of this change to focus on the Qt fix. I can create another PR to support this and update all the current projects in the future.

Reply:
That sounds good to me.

Reply:
GHI: https://github.com/o3de/3p-package-source/issues/217